### PR TITLE
UIBULKED-461: Basic  UI only data entry validation on the MARC instan…

### PR DESCRIPTION
**Pull request by UladzislauKutarkin**
https://folio-org.atlassian.net/browse/UIBULKED-461
Here we added `UI` validation for `Marc` form

https://github.com/folio-org/ui-bulk-edit/assets/72550466/edc5d796-7896-4c20-a500-484966d2d4ad

### CODE REVIEW

File: src/components/BulkEditPane/BulkEditListResult/BulkEditMark/BulkEditMarkForm/BulkEditMarkForm.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ import {
   getDefaultMarkTemplate,
   TAG_FIELD_MAX_LENGTH,
   INDICATOR_FIELD_MAX_LENGTH,
-  SUBFIELD_MAX_LENGTH,
+  SUBFIELD_MAX_LENGTH, isMarcValueValid,
```
Comment (vashjs) [inline src/components/BulkEditPane/BulkEditListResult/BulkEditMark/BulkEditMarkForm/BulkEditMarkForm.js:15]: pleae move on the next line

File: src/components/BulkEditPane/BulkEditListResult/BulkEditMark/BulkEditMarkForm/BulkEditMarkForm.js [hunk 2]
```diff
@@ -29,6 +29,18 @@ const BulkEditMarkForm = () => {
 
   const isIndicatorDirty = (value) => value?.length && value !== '\\';
 
+  const isValueFieldValid = (value) => {
+    if (Number(value.trim()).toString().length === 3) {
+      return isMarcValueValid(value);
+    } else return true;
```
Comment (vashjs) [inline src/components/BulkEditPane/BulkEditListResult/BulkEditMark/BulkEditMarkForm/BulkEditMarkForm.js:35]: else is redundant, can be just 

`return true;`

File: src/components/BulkEditPane/BulkEditListResult/BulkEditMark/helpers.js [hunk 3]
```diff
@@ -117,9 +117,21 @@ export const getNextDataControls = (action) => {
   }
 };
 
+export const isMarcValueValid = (value) => {
```
Comment (vashjs) [inline src/components/BulkEditPane/BulkEditListResult/BulkEditMark/helpers.js:120]: typo in name, should be `isMarkValueValid`

**Review by vashjs — CHANGES_REQUESTED** [review]

**Comment by vashjs** [comment]
Please increase test coverage to 80+

File: src/components/BulkEditPane/BulkEditListResult/BulkEditMark/helpers.js (hunk 3 above)
Comment (UladzislauKutarkin) [reply src/components/BulkEditPane/BulkEditListResult/BulkEditMark/helpers.js]: done!

File: src/components/BulkEditPane/BulkEditListResult/BulkEditMark/BulkEditMarkForm/BulkEditMarkForm.js (hunk 1 above)
Comment (UladzislauKutarkin) [reply src/components/BulkEditPane/BulkEditListResult/BulkEditMark/BulkEditMarkForm/BulkEditMarkForm.js]: done

File: src/components/BulkEditPane/BulkEditListResult/BulkEditMark/BulkEditMarkForm/BulkEditMarkForm.js (hunk 2 above)
Comment (UladzislauKutarkin) [reply src/components/BulkEditPane/BulkEditListResult/BulkEditMark/BulkEditMarkForm/BulkEditMarkForm.js]: done

File: src/components/BulkEditPane/BulkEditMarkLayer/BulkEditMarkLayer.test.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { MemoryRouter } from 'react-router-dom';
 import { QueryClientProvider } from 'react-query';
-import { render, waitFor, within } from '@testing-library/react';
+import { render, waitFor, within, screen, getByTestId, logDOM } from '@testing-library/react';
```
Comment (artem-blazhko) [inline src/components/BulkEditPane/BulkEditMarkLayer/BulkEditMarkLayer.test.js:4]: Could you remove unused methods (getByTestId, logDOM)?

Comment (UladzislauKutarkin) [reply src/components/BulkEditPane/BulkEditMarkLayer/BulkEditMarkLayer.test.js]: Sure, done! @artem-blazhko please check it out again

**Comment by Dmitriy-Litvinenko** [comment]
Hello @artem-blazhko @vashjs Could you please review?

**Review by artem-blazhko — APPROVED** [review]

**Review by vashjs — APPROVED** [review]

**Pull request merged**